proto/message_contents/invitation.proto

```diff
@@ -0,0 +1,43 @@
+// Signature
+syntax = "proto3";
+
+package xmtp.message_contents;
+
+import "message_contents/ciphertext.proto";
+import "message_contents/public_key.proto";
+
+option go_package = "github.com/xmtp/proto/go/message_contents";
+
+// Unsealed invitation V1
+message InvitationV1 {
+    bytes topic = 1; // randomly generated topic name (>= 32 bytes)
```

Comment:
This reflects the proposal from the Key Passing RFC.

Reply:
Given that the client SDK treats topics as strings everywhere, it would be helpful for this to also be a string. Obviously we can base64 encode it when we use it later, but it seems more straightforward to handle that encoding/decoding once at the time of creation.

Reply:
OK, so the topic will be generated as 32 random bytes and then base64 encoded (unpadded). Sound good?

What is the easiest thing to do for the message signature? Just take the ascii bytes to concatenate with the header and payload?

Reply:
That sounds reasonable